forte/data/readers/classification_reader.py

```diff
@@ -0,0 +1,260 @@
+# Copyright 2020 The Forte Authors. All Rights Reserved.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#      http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+import csv
+import importlib
+from typing import Iterator, Tuple, List, Dict
+from forte.common import Resources, ProcessorConfigError
+from forte.common.configuration import Config
+from forte.data.data_pack import DataPack
+from forte.data.base_reader import PackReader
+from ft.onto.base_ontology import *
+
+__all__ = ["ClassificationDatasetReader"]
+
+
+class ClassificationDatasetReader(PackReader):
+    r""":class:`ClassificationDatasetReader` is designed to read table-like
+    classification datasets that contain input text and digit/text labels.
+    There are a couple of values that need to be provided via configuration,
+    including `forte_data_fields`, `text_fields`, and `index2class`.
+
+    `forte_data_fields` is a list representing the column headers of the
+    dataset. Each element in the `forte_data_fields` list is either a label
+    that indicates the body class or a Forte entry type that will be used
+    to store the content. Apparently, the list should follow the column order
+    of the dataset.
+
+    For example, for amazon polarity dataset,
+    (https://huggingface.co/datasets/amazon_polarity), the column names
+    are [label, title, content]. We can configure `forte_data_fields`to be
+    ['label', 'ft.onto.base_ontology.Title', 'ft.onto.base.Body'].
```

Comment:
This "ft.onto.base.Body" is also wrong, please be careful of what you wrote, too much wrong typing here and there.

Reply:
Please resolve this

Reply:
Fixed.